touchforms/backend/persistence.py

```diff
@@ -45,11 +46,51 @@ def cache_get(key):
 
 
 def cache_del(key):
-    raise RuntimeError('not implemented')
-
-def cache_path(key):
-    # todo: make this use something other than the filesystem
-    persistence_dir = settings.PERSISTENCE_DIRECTORY or tempfile.gettempdir()
-    if not os.path.exists(persistence_dir):
-        os.makedirs(persistence_dir)
-    return os.path.join(persistence_dir, 'tfsess-%s' % key)
+    conn = get_conn()
+    cursor = conn.cursor()
+    delete_query = "DELETE FROM " + settings.POSTGRES_TABLE + " WHERE sess_id='" + key + "'"
+    cursor.execute(delete_query)
+    conn.commit()
+    conn.close()
+
+
+def postgres_lookup(key):
+    conn = get_conn()
+    cursor = conn.cursor()
+    query = "SELECT sess_json FROM " + settings.POSTGRES_TABLE + " WHERE sess_id='" + key + "'"
+    cursor.execute(query)
+    value = cursor.fetchone()[0].getValue()[1:-1]   # yeah... this is terrible. We get the Row -> PGObject ->
+                                                    # Unicode -> Strip [] so that json will convert to dict
```

Comment:
it is relevant... maybe I need to explain better? I basically just have to dig the value I need out of a lot of object layers, trying to explain that